addons/l10n_ar/views/report_invoice.xml

```diff
@@ -222,7 +222,13 @@
                     <!-- We do not have remitos implement yet. print here the remito number when we have it -->
 
                     <t t-if="o.invoice_incoterm_id">
-                        <br/><strong>Incoterm:</strong><span t-field="o.invoice_incoterm_id.name"/>
+                        <br/>
+                        <strong>Incoterm:</strong>
+                         <p t-if="o.incoterm_location">
+                            <span t-field="o.invoice_incoterm_id.code"/> <br/>
+                            <span t-field="o.incoterm_location"/>
+                        </p>
+                        <p t-else="" t-field="o.invoice_incoterm_id.name" class="m-0"/>
```

Comment:
For someone who is tilted about indentation on other PRs... :sweat_smile:

Reply:
erkk didn't see these ones :laughing: thx will change that !